plugins/keyboard-navigation/src/navigation_controller.js

```diff
@@ -113,7 +113,8 @@ export class NavigationController {
           return false;
       }
     }
-    return Blockly.FieldColour.superClass_.onShortcut.call(this, shortcut);
+    // Field is the superclass of FieldColour.
```

Comment:
nit, but outside the context of this code review where i can see the line you're removing, this comment doesn't help me understand what's going on.

"If we haven't already handled the shortcut, let the default Field handler try" or something might be more clear. same below.

Reply:
Done.